exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+var name1 = "Lana";
+var nameLength = name1.length;
+var message = "My name is " + name1 + " and my name is " + nameLength + " characters long";
```

Comment:
great work. You could maybe try next time to write it using the following syntax:
`My name is ${name1} and my name is ${nameLength} characters long`